src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,7 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          => // TODO do nothing; this needs to be checked in the preconditions
```

Comment:
I suggest the following: we throw an `IllegalArgumentException` here as this case indeed should never occur. However, I prefer to keep this TODO in the code for now as a reminder to the one who is going to fix issue #15. Let's also add a notice about this TODO in the Jira issue.